R/dygraph.R

```diff
@@ -70,16 +93,17 @@ dygraph <- function(data, main = NULL, xlab = NULL, ylab = NULL,
   # create x (dygraph attrs + some side data)
   x <- list()
   x$attrs <- attrs
-  x$scale <- periodicity$scale
+  x$scale <- if (format == "date") periodicity$scale else NULL
```

Comment:
I noticed another use of `x.scale` in dygraphs.js that appears to still be in play even format = 'numeric'. https://github.com/dgromer/dygraphs/blob/master/inst/htmlwidgets/dygraphs.js#L524   Does this seem like an issue?